### PR TITLE
Review and fix section references into other QUIC docs.

### LINKED ISSUE #100: verify all references to QUIC documents
The ops drafts often reference the core drafts by section number; make sure these section number references are correct for the WGLC drafts.

### PR DESCRIPTION
**Pull request by britram**
fixes #100

### CODE REVIEW

**Review by mirjak — APPROVED** [review]

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -319,9 +319,9 @@ of data to streams dependent on this information.
 
 While a QUIC implementation must necessarily provide a way for an application
 to send data on separate streams, it does not necessarily expose stream
-identifiers to the application (see e.g. {{QUIC-HTTP}} section 6) either at
-the sender or receiver end, so applications should not assume access to these
-identifiers.
+identifiers to the application (see, for example, {{QUIC-HTTP}} section 6)
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:322]:
```suggestion
identifiers to the application (see, for example, {{QUIC-HTTP}}, Section 6)
```

File: draft-ietf-quic-applicability.md [hunk 2]
```diff
@@ -518,13 +518,13 @@ therefore no additional data is expected from both sides.
...
 of types: one for QUIC internal problems that might lead to connection closure,
 and one for closures initiated by the application. An application using QUIC can
-define application-specific error codes, e.g. see {{QUIC-HTTP}} section 8.1. In
-the case of a grateful shut-down initiated by the application after application
-layer negotiation, a NO_ERROR code is expected. Further, the CONNECTION_CLOSE
-frame provides an optional reason field, that can be used to append
-human-readable information to an error code. Note that QUIC RESET_STREAM and
-STOP_SENDING frames provide similar capablities. Usually application error codes
-are defined to be applicabile to all three frames.
+define application-specific error codes (see, for example, {{QUIC-HTTP}}
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:521]:
```suggestion
define application-specific error codes (see, for example, {{QUIC-HTTP}},
```

File: draft-ietf-quic-applicability.md [hunk 3]
```diff
@@ -518,13 +518,13 @@ therefore no additional data is expected from both sides.
...
+section 8.1). In the case of a grateful shut-down initiated by the application
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:522]:
```suggestion
Section 8.1). In the case of a grateful shut-down initiated by the application
```

File: draft-ietf-quic-applicability.md [hunk 4]
```diff
@@ -185,7 +185,7 @@ Data sent during 0-RTT resumption also cannot benefit from perfect forward
 secrecy (PFS).
 
 Data in the first flight sent by the client in a connection established with
-0-RTT MUST be idempotent (as specified in section 2.1 in {{!QUIC-TLS}}).
+0-RTT MUST be idempotent (as specified in section 5.6 in {{!QUIC-TLS}}).
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:188]:
```suggestion
0-RTT MUST be idempotent (as specified in Section 5.6 in {{!QUIC-TLS}}).
```

**Review by martinthomson — APPROVED** [review]
Correcting overcorrections regarding capitalization, but this seems benign.

**Pull request merged**